api/client.js

```diff
@@ -0,0 +1,34 @@
+const proxyurl = ""
+
+const prefix =
+  process.env.NODE_ENV === 'production'
+    ? 'https://www.reflectionsprojections.org'
+    : 'http://localhost:3000';  
+
+export const fetchConferenceData = async () => {
+  const url = `${prefix}/data/rp2020.json`;
```

Comment:
this isnt the right way to access data...in next u can access anything iin the public folder just using /blah.json

Reply:
Noted, thanks for pointing this out. I removed the usage of 'prefix'